src/Nowruz/modules/orgSettings/components/team/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { Icon } from "src/Nowruz/general/Icon";
+import { Avatar } from "src/Nowruz/modules/general/components/avatar/avatar";
+import { Button } from "src/Nowruz/modules/general/components/Button";
+import { SearchDropdown } from "src/Nowruz/modules/general/components/SearchDropdown";
+
+import css from "./team.module.scss";
+import { useTeam } from "./useTeam";
+
+
+const Team = () => {
+    const {members,onRemove,memberList,onSearchMember,onAddMember,role,userId} = useTeam();
+    return(
+        <>
+            <div className={css.borderSection}>
+                <div className="w-full py-8 items-center">
+                    <h2 className="grow css.title text-lg font-semibold">Team management</h2>
+                    <p className='text-sm font-normal text-Gray-light-mode-600 pt-1'>
+                    Manage your team members and their account permissions here.  
+                    </p>     
+                </div>
+            </div>
+
+            <div className="grid grid-cols-5 gap-4 pt-6">
+                <div>
+                    <h6 className="text-sm font-semibold text-Gray-light-mode-700">Invite team members</h6>
+                    <p className="text-sm font-normal text-Gray-light-mode-600">
+                    Invite member to post or manage your organization
+                    </p>
+                </div>
+                <div className="col-span-3">
+                    <SearchDropdown
+                    placeholder="Please select"
+                    icon="image-user"
+                    options={memberList}
+                    isSearchable
+                    onChange={(option) => onSearchMember(option.value)}
+                    />
+                </div>
+                <div>
+                    <SearchDropdown
+                        placeholder="Please select"
+                        options={role}
+                        isSearchable
+                    />
+                </div>
+            </div>
+            <div className="grid grid-cols-5 gap-4 align-bottom mt-4">
+                <div>&nbsp;</div>
```

Comment:
no need to add additional element for just space

Reply:
This page is old and removed totally, after merging with main branch we won't have this page anymore